lib/krikri.rb

```diff
@@ -12,4 +12,5 @@ module Krikri
   autoload :OaiDcParser,    'krikri/parsers/oai_dc_parser'
   autoload :JsonParser,     'krikri/parsers/json_parser'
   autoload :ModsParser,     'krikri/parsers/mods_parser'
+  autoload :QdcParser,      'krikri/parsers/qdc_parser'
 end
```

Comment:
Oh... the line is `config.autoload_paths += Dir["#{config.root}/lib/**/"]` not `.../**/*`.

It's not including subdirs. LDP and Enrichments are requiring everything below them, but Parsers is not.